src/OrleansBondUtils/BondSerializer.cs

```diff
@@ -191,15 +197,35 @@ private void Register(Type type)
             var clonerType = typeof(Cloner<>);
             var realType = clonerType.MakeGenericType(type);
             var clonerInstance = Activator.CreateInstance(realType);
-            var cloneMethod = realType.GetMethod("Clone").MakeGenericMethod(type);
-            var copierDelegate = cloneMethod.CreateDelegate(
-                    typeof(Func<,>).MakeGenericType(new[] { type, type }),
-                    clonerInstance);
             var serializer = new BondTypeSerializer(type);
             var deserializer = new BondTypeDeserializer(type);
-            CopierDictionary.TryAdd(type.TypeHandle, copierDelegate);
+            var parameterExpression1 = Expression.Parameter(typeof(object));
```

Comment:
these names should be more descriptive, eg: 'instanceParameter` and `sourceParameter`